src/styles/blocks/watch.scss

```
@@ -0,0 +1,60 @@
+.stopwatch {
+  @include clockSize;
+
+  position: relative;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  background-color: #b7cdc9;
+
+  border: 3px solid #0f1312;
+  border-radius: 50%;
+
+  box-shadow: 20px 20px 20px #999;
```

Maybe you don't need this empty lines)
```suggestion
  background-color: #b7cdc9;
  border: 3px solid #0f1312;
  border-radius: 50%;
  box-shadow: 20px 20px 20px #999;
```